samza-core/src/main/scala/org/apache/samza/job/local/ThreadJobFactory.scala

```diff
@@ -147,7 +150,6 @@ class ThreadJobFactory extends StreamJobFactory with Logging {
       if (jmxServer != null) {
         jmxServer.stop
       }
-      coordinatorStreamStore.close()
```

Comment:
Cannot stop the coordinatorStreamStore in `finally()` as it gets used by the container.